### PR TITLE
Responsive pages 2

### PR DESCRIPTION
**Pull request by d-elv**
Fingers crossed this addresses the responsive web design issue!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.20.1",
     "react-scripts": "^5.0.1",
+    "vite": "^5.1.1",
```
Comment (ChristienGuy) [inline package.json]: nit: How come we're adding a bundler in a styling update PR?

It might be worth reviewing your own PR's before asking others to do so, so that you can catch things like this and either:

1. Explain why the seemingly strange addition is necessary
2. Remove the strange addition

**Review by ChristienGuy — APPROVED** [review]
Lookin dope! 

I've got a nit comment about a dependency change that doesn't make sense to me but it's not anything I'd block a merge on. 

Great work 🙏

File: package.json (hunk 1 above)
Comment (d-elv) [reply package.json]: I have now removed this dependency change as it wasn't needed! Thank you for picking up on it and noted about self-reviewing, that's a good idea. Makes sense to aid not rushing things.
